python/arcticdb/version_store/_normalization.py

```diff
@@ -807,9 +807,9 @@ class MsgPackNormalizer(Normalizer):
     Fall back plan for the time being to store arbitrary data
     """
 
-    MMAP_DEFAULT_SIZE = 1 << 32  # Allow up to 4 gib pickles in memory, most of these compress fairly well.
+    MMAP_DEFAULT_SIZE = (1 << 32) - 1  # Allow up to 4 gib pickles in memory, most of these compress fairly well.
```

Comment:
The comment is no longer relevant. The limitation is from the library, not our design choice.

Reply:
Comment updated.
I have considered removing the variables but doing so may break backward compatibility.